unified-codes/apps/web/src/components/templates/Details.tsx

```diff
@@ -2,41 +2,107 @@ import * as React from 'react';
 import { Dispatch } from 'redux';
 import { connect } from 'react-redux';
 import { useParams } from 'react-router-dom';
-import { IEntity, } from '@unified-codes/data';
+
+import { IEntity, Property } from '@unified-codes/data';
+import {
+  EntityDetailList,
+  IExternalLink,
+  Grid,
+  Typography,
+  CircularProgress,
+  Backdrop,
+} from '@unified-codes/ui';
+
 import { DetailsActions } from '../../actions';
 import { IState } from '../../types';
+import { ITheme } from '../../muiTheme';
+
+import { withStyles } from '@material-ui/core/styles';
+import { ClassNameMap } from '@material-ui/core/styles/withStyles';
 
 export interface DetailsProps {
- getEntity: (code: string) => void;
- entity?: IEntity
+  getEntity: (code: string) => void;
+  entity?: IEntity;
+  classes: ClassNameMap<any>;
+  isLoading: boolean;
 }
 
+const getStyles = (theme: ITheme) => ({
+  backdrop: {
+    zIndex: theme.zIndex.drawer + 1,
+    color: '#fff',
+  },
+  root: {
+    paddingLeft: 20,
+    paddingTop: 20
+  }
+});
+
 interface IDetailsParameters {
   code: string;
 }
 
 export type Details = React.FunctionComponent<DetailsProps>;
 
-export const DetailsComponent: Details = ({ entity, getEntity }) => {
+export const DetailsComponent: Details = ({ classes, entity, isLoading, getEntity }) => {
   const { code } = useParams<IDetailsParameters>();
 
   React.useEffect(() => {
     getEntity(code);
   }, []);
 
-  // TODO: Create UI components and format this properly
-  return <div>{JSON.stringify(entity)}</div>;
+  let productSubCategories: IEntity[] = [];
```

Comment:
these can be `const` as they aren't reassigned. The array has items pushed into it, that's all.

Reply:
Oh - thanks, that's a good point. I must have been thinking of a different kind of array at the time. Updated! 63e5bdf